### DISCUSSION TITLE
New User Interface

### DISCUSSION THREAD

**Discussion by ItsMarzouq** [SpotX-Official/SpotX#76] [General]
The new **UI** for Block the Spot is a bit scary.
Previous was more generic and worked with only one click.
Almost a bit difficult to trust. 
But works fine. 
Thanks!

**Comment by amd64fox** [comment]
So what's the problem ? You can continue to use the [old BTS patch](https://github.com/mrpond/BlockTheSpot/tree/snapshot-pre-spotx-merge), it has not been removed.

**Comment by ItsMarzouq** [comment]
Oh hi there. I would really like that but it's not working for me. All the
codes in the first few lines have turned red.

**Reply by amd64fox** [reply]
This works if you run the script from the page I indicated above, a little more carefully.

**Comment by ItsMarzouq** [comment]
Hey thank you so much! I'll look into it. BTW amazing code you have made,
life saver for sure man!
